frontend/src/insecure/AntiPatternList.jsx

```
@@ -0,0 +1,13 @@
+/**
+ * Missing keys in list items; inline handlers allocated per render.
+ */
+import React from "react";
+
+export default function AntiPatternList(){
+  const arr = Array.from({length:20}, (_,i)=>({i, v: Math.random()}));
+  return (
+    <ol>
+      {arr.map(x => <li onClick={()=>console.log(x)}>{x.v}</li>)}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)